src/main/java/com/olgaboiar/guacomole/ServerRunner.java

```diff
@@ -0,0 +1,11 @@
+package com.olgaboiar.guacomole;
+
+import java.io.IOException;
+
+public class ServerRunner {
+    public static void main(String[] args) throws IOException {
+        EchoServer server;
+        server = new EchoServer();
+        server.start("localhost", 3333);
```

Comment:
Can we make the host and port passed in through command line arguments?

Reply:
implemented in https://github.com/olgaboiar/GuacamoleServer/pull/2/commits/f8ca23456d33f499d56b2a558f170126593b5df3